### PR TITLE
ref(ci): bump sentry in getsentry directly

### PR DESCRIPTION
**Pull request by joshuarli**
Should also obviate the need for https://github.com/getsentry/getsentry/pull/7971.

If this works (we'll know when it merges), I'm gonna f/u deleting this in getsentry.

### CODE REVIEW

File: .github/workflows/bump-sentry-in-getsentry.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+      - master
+
+jobs:
+  bump-sentry:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: checkout
+        uses: actions/checkout@d0651293c4a5a52e711f25b41b05b2212f385d28  # v3
+        with:
+          # For bin/bump-sentry, sentry needs to be at ../sentry relative to getsentry.
```
Comment (evanpurkhiser) [inline .github/workflows/bump-sentry-in-getsentry.yml:19]:
```suggestion
          # For getsentry/bin/bump-sentry, sentry needs to be at ../sentry relative to getsentry.
```

File: .github/workflows/bump-sentry-in-getsentry.yml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+          path: sentry
+
+      - name: checkout getsentry
+        uses: actions/checkout@d0651293c4a5a52e711f25b41b05b2212f385d28  # v3
+        with:
+          repository: 'getsentry/getsentry'
+          path: getsentry
+          # This PAT (Personal Access Token) belongs to getsentry-bot,
```
Comment (evanpurkhiser) [inline .github/workflows/bump-sentry-in-getsentry.yml:27]:
```suggestion
          # This Personal Access Token belongs to getsentry-bot,
```

nit: I would use the acronym if you're gonna say it again later in this file, but we don't

File: .github/workflows/bump-sentry-in-getsentry.yml [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+          # who can write to getsentry and is SAML+SSO ready.
+          token: ${{ secrets.BUMP_SENTRY_TOKEN }}
+
+      - name: install dependencies
+        shell: bash
+        run: |
+          cd getsentry
+          python -m pip install -q "$(grep '^pip-tools==' requirements-dev-frozen.txt)"
+
+      - name: bump-sentry ${{ inputs.sentry_sha }}
```
Comment (evanpurkhiser) [inline .github/workflows/bump-sentry-in-getsentry.yml:37]: Is it still an input? won't this be this now
```suggestion
      - name: bump-sentry ${{ github.sha }}
```

Comment (joshuarli) [reply .github/workflows/bump-sentry-in-getsentry.yml:37]: Oh, yeah, forgot to change this part.

**Review by evanpurkhiser — APPROVED** [review]

**Review by asottile-sentry — APPROVED** [review]

**Pull request merged**